proto/provenance/ledger/v1/tx.proto

```diff
@@ -0,0 +1,298 @@
+syntax = "proto3";
+package provenance.ledger.v1;
+
+option go_package          = "github.com/provenance-io/provenance/x/ledger/types";
+option java_package        = "io.provenance.ledger.v1";
+option java_multiple_files = true;
+
+import "cosmos/msg/v1/msg.proto";
+import "provenance/ledger/v1/ledger.proto";
+import "provenance/ledger/v1/ledger_settlement.proto";
+
+// Msg defines the ledger module Msg service.
+service Msg {
+  option (cosmos.msg.v1.service) = true;
+
+  // Create a new NFT ledger
+  rpc Create(MsgCreateRequest) returns (MsgCreateResponse);
+
+  // Update Status
+  rpc UpdateStatus(MsgUpdateStatusRequest) returns (MsgUpdateStatusResponse);
+
+  // Update Interest Rate
+  rpc UpdateInterestRate(MsgUpdateInterestRateRequest) returns (MsgUpdateInterestRateResponse);
+
+  // Update Payment Amount, Next Payment Date, and Payment Frequency
+  rpc UpdatePayment(MsgUpdatePaymentRequest) returns (MsgUpdatePaymentResponse);
+
+  // Update Maturity Date
+  rpc UpdateMaturityDate(MsgUpdateMaturityDateRequest) returns (MsgUpdateMaturityDateResponse);
+
+  // Append a ledger entry
+  rpc Append(MsgAppendRequest) returns (MsgAppendResponse);
+
+  // Balances can be updated for a ledger entry allowing for retroactive adjustments to be applied
+  rpc UpdateBalances(MsgUpdateBalancesRequest) returns (MsgUpdateBalancesResponse);
+
+  // Process multiple fund transfers with manual settlement instructions
+  rpc TransferFundsWithSettlement(MsgTransferFundsWithSettlementRequest)
+      returns (MsgTransferFundsWithSettlementResponse);
+
+  // Destroy a ledger by NFT address
+  rpc Destroy(MsgDestroyRequest) returns (MsgDestroyResponse);
+
+  // Create a new ledger class
+  rpc CreateLedgerClass(MsgCreateLedgerClassRequest) returns (MsgCreateLedgerClassResponse);
+
+  // Add a status type to a ledger class
+  rpc AddLedgerClassStatusType(MsgAddLedgerClassStatusTypeRequest) returns (MsgAddLedgerClassStatusTypeResponse);
+
+  // Add an entry type to a ledger class
+  rpc AddLedgerClassEntryType(MsgAddLedgerClassEntryTypeRequest) returns (MsgAddLedgerClassEntryTypeResponse);
+
+  // Add a bucket type to a ledger class
+  rpc AddLedgerClassBucketType(MsgAddLedgerClassBucketTypeRequest) returns (MsgAddLedgerClassBucketTypeResponse);
+
+  // Bulk create ledgers and entries
+  rpc BulkCreate(MsgBulkCreateRequest) returns (MsgBulkCreateResponse);
+}
+
+// MsgCreateRequest represents a request to create a new ledger.
+message MsgCreateRequest {
```

Comment:
Suggest rename to `MsgCreateLedgerRequest` to be consistent with other objection creation messages.